kroxylicious-operator-dist/pom.xml

```diff
@@ -28,12 +28,6 @@
             <version>${project.version}</version>
             <scope>provided</scope>
         </dependency>
-        <dependency>
-            <groupId>io.kroxylicious</groupId>
-            <artifactId>kroxylicious-docs</artifactId>
-            <version>${project.version}</version>
-            <scope>provided</scope>
-        </dependency>
```

Comment:
I think we need this to ensure the docs are built before `kroxylicious-operator-dist` assembles its tarball and pulls docs in.

Reply:
would be good if we could get the operator dist build to fail fast instead of silently not incorporating a doc directory

Reply:
Readded.  Its removal was unintentional.

> would be good if we could get the operator dist build to fail fast instead of silently not incorporating a doc directory

I don't think we have a way to express that in Maven